org-formation/600-access/_tasks.yaml

```diff
@@ -388,3 +388,39 @@ SynapseAthenaUserAccessPolicy:
         ]
       }
     PolicyName: SynapseAthenaUserAccessPolicy
+
+SynapseLlmDeveloperPolicy:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/IAM/managed-policy.yaml
+  StackName: synapsellm-developer-policy
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account:
+      - SynapseLlmProdAccount
+    Region: !Ref primaryRegion
+  Parameters:
+    PolicyDocument: >-
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": "s3:*",
+            "Resource": "arn:aws:s3:::cf-template*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": "iam:PassRole",
+            "Resource": "*"
+          }
+        ]
+      }
+    PolicyName: SynapseLlmDeveloperPolicy
+
+BedrockAgentRole:
```

Comment:
why is this role necessary when the goal is to only fix the developer role?

Reply:
This is the policy for the user (sso-llmdeveloper) to be able to work in the console/cloudformation to create an agent.